redash/utils/parameterized_query.py

```diff
@@ -16,19 +16,22 @@ def _pluck_name_and_value(default_column, row):
     return {"name": row[name_column], "value": row[value_column]}
```

Comment:
(unrelated to this line)

I think this class should move into `redash.models`.

Reply:
3e8d1a51